tests/e2e/test_create_movies_page.py

```diff
@@ -1 +1,25 @@
 # TODO: Feature 2
+import pytest
+from app import app, create_movie
+
+@pytest.fixture()
+def client():
+    client = create_movie('movie_title','movie_director','1')
```

Comment:
You cant call create_movie without its respective object. I'm not sure I understand everything your doing in this function but it doesnt seem like your passing in html data into your app. Calling movie_repository.create_movie will bypass your function and therefore bypass the thing you are testing